lib/worldwide/field.rb

```diff
@@ -48,6 +48,9 @@ def label_marked_optional(locale: I18n.locale)
     end
 
     def error(locale: I18n.locale, code:, options: {})
+      unless code.end_with?("_instructional") || code.end_with?("_informative")
```

Comment:
Reverts logic of this back to https://github.com/Shopify/worldwide/blob/7e4a6892ba83d2183ce811c4563877b2584d46aa/lib/worldwide/field.rb

If a error message key doesn't end with either `_instructional` or `_informative` we will default it to the `_instructional`.